recipes/FC-Virus/mete.yaml

```diff
@@ -0,0 +1,32 @@
+package:
```

Comment:
@qdu-bioinfo  Please rename the file from `mete.yaml` to `meta.yaml`

Reply:
Thank you for your reply, I have renamed it.